src/__test__/hdwallet.spec.ts

```diff
@@ -146,6 +147,7 @@ describe("HD wallet test", () => {
             "11fca78e1985e951c9cc7a866626741cefe972a24dd1b68a5c53940436b33a026e0c4736b6b6bc4b3b50600cf8c780e4ca01772272d3664939c571d29fa5a1d800"
         ]
     ])(
+        // FIXME
```

Comment:
I don't know how to make a test oracle like the example above.

Reply:
@HoOngEe Could you give me good advice? Can I create a testing oracle about this in the rust implementation?

Reply:
It is much more simple in this case, you can expect each function's behavior by reading the code and generate oracles from python implementation of ed25519 or our primitives.